README.rst

```diff
@@ -1874,823 +67,14 @@ as in the example below:
-don't reinvent the wheel.
-
-The function :code:`insert_arguments_into_query` does so based on the query
-language, which isn't fine-grained enough here-- for Cypher backends, we
-only want to insert parameters if the backend is RedisGraph, but not if
-it's Neo4j.
-
-Instead, the correct approach for Neo4j Cypher is as follows, given a
-Neo4j Python client called :code:`neo4j_client`:
-
-.. code:: python
-
-    common_schema_info = CommonSchemaInfo(schema, type_equivalence_hints)
-    compilation_result = compile_graphql_to_cypher(common_schema_info, graphql_query)
-    with neo4j_client.driver.session() as session:
-        result = session.run(compilation_result.query, parameters)
-
-FAQ
----
-
-**Q: Do you really use GraphQL, or do you just use GraphQL-like
-syntax?**
-
-A: We really use GraphQL. Any query that the compiler will accept is
-entirely valid GraphQL, and we actually use the Python port of the
-GraphQL core library for parsing and type checking. However, since the
-database queries produced by compiling GraphQL are subject to the
-limitations of the database system they run on, our execution model is
-somewhat different compared to the one described in the standard GraphQL
-specification. See the `Execution model <#execution-model>`__ section
-for more details.
-
-**Q: Does this project come with a GraphQL server implementation?**
-
-A: No -- there are many existing frameworks for running a web server. We
-simply built a tool that takes GraphQL query strings (and their
-parameters) and returns a query string you can use with your database.
-The compiler does not execute the query string against the database, nor
-does it deserialize the results. Therefore, it is agnostic to the choice
-of server framework and database client library used.
-
-**Q: Do you plan to support other databases / more GraphQL features in
-the future?**
-
-A: We'd love to, and we could really use your help! Please consider
-contributing to this project by opening issues, opening pull requests,
-or participating in discussions.
-
-**Q: I think I found a bug, what do I do?**
-
-A: Please check if an issue has already been created for the bug, and
-open a new one if not. Make sure to describe the bug in as much detail
-as possible, including any stack traces or error messages you may have
-seen, which database you're using, and what query you compiled.
-
-**Q: I think I found a security vulnerability, what do I do?**
-
-A: Please reach out to us at graphql-compiler-maintainer@kensho.com so
-we can triage the issue and take appropriate action.
+.. end-to-end-sql-example-end
 
 License
 -------
 
-Licensed under the Apache 2.0 License. Unless required by applicable law
-or agreed to in writing, software distributed under the License is
-distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-KIND, either express or implied. See the License for the specific
-language governing permissions and limitations under the License.
```

Comment:
Since we are making the README file shorter, I thought it would be worthwhile improve the content-to-boiler plate ratio by adding a link to the license instead of having this text here. (This was ReadTheDocs does on their github page). Thoughts?

Reply:
It seems that this is a no-go. I am going to undo this.

Reply:
Reminder to undo this